go/tasks/plugins/array/k8s/config.go

```diff
@@ -113,8 +115,14 @@ type Config struct {
 	RemoteClusterConfig  ClusterConfig     `json:"remoteClusterConfig" pflag:"-,Configuration of remote K8s cluster for array jobs"`
 	NodeSelector         map[string]string `json:"node-selector" pflag:"-,Defines a set of node selector labels to add to the pod."`
 	Tolerations          []v1.Toleration   `json:"tolerations"  pflag:"-,Tolerations to be applied for k8s-array pods"`
+	Namespace            string            `json:"namespace"  pflag:"-,Namespace pattern to spawn array-jobs in. Defaults to parent namespace if not set"`
```

Comment:
I don't think this is right. This is config for the entire deployment right? Not for workflow

Reply:
Yes, this is for deployment and namespace here is a regex like `"a-{{.namespace}}-b"` so platform admin can decide on this if they want to use a separate namespace to run array-jobs. If this is not set, it's ignored and current behavior is maintained.

Reply:
resolving, as discussed